console/console-init/ui/src/stories/UpdatePassword.stories.tsx

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React from "react";
+import { HashRouter as Router } from "react-router-dom";
```

Comment:
Is there any reason of not using MemoryRouter here, as used in other stories.

Reply:
There is no specific reason here. We can use anyone of them. Just to avoid router issue in storybook.

Reply:
okay